tests/integration-service/status-reporting-to-pullrequest.go

```diff
@@ -75,23 +80,24 @@ var _ = framework.IntegrationServiceSuiteDescribe("Status Reporting of Integrati
 			}, timeout, constants.PipelineRunPollingInterval).Should(Succeed(), fmt.Sprintf("timed out when waiting for the build PipelineRun to start for the component %s/%s", testNamespace, componentName))
 			labels = pipelineRun.GetLabels()
 			annotations = pipelineRun.GetAnnotations()
+			fmt.Print(componentBaseBranchName)
 		})
 
-		AfterAll(func() {
-			if !CurrentSpecReport().Failed() {
-				cleanup(*f, testNamespace, applicationName, componentName, snapshot)
-			}
-
-			// Delete new branches created by PaC and a testing branch used as a component's base branch
-			err = f.AsKubeAdmin.CommonController.Github.DeleteRef(componentRepoNameForStatusReporting, pacBranchName)
-			if err != nil {
-				Expect(err.Error()).To(ContainSubstring(referenceDoesntExist))
-			}
-			err = f.AsKubeAdmin.CommonController.Github.DeleteRef(componentRepoNameForStatusReporting, componentBaseBranchName)
-			if err != nil {
-				Expect(err.Error()).To(ContainSubstring(referenceDoesntExist))
-			}
-		})
+		// AfterAll(func() {
+		// 	if !CurrentSpecReport().Failed() {
+		// 		cleanup(*f, testNamespace, applicationName, componentName, snapshot)
+		// 	}
+
+		// 	// Delete new branches created by PaC and a testing branch used as a component's base branch
+		// 	err = f.AsKubeAdmin.CommonController.Github.DeleteRef(componentRepoNameForStatusReporting, pacBranchName)
+		// 	if err != nil {
+		// 		Expect(err.Error()).To(ContainSubstring(referenceDoesntExist))
+		// 	}
+		// 	err = f.AsKubeAdmin.CommonController.Github.DeleteRef(componentRepoNameForStatusReporting, componentBaseBranchName)
+		// 	if err != nil {
+		// 		Expect(err.Error()).To(ContainSubstring(referenceDoesntExist))
+		// 	}
+		// })
```

Comment:
Missed this also - should it be commented?

Reply:
fixed